Mathlib/FieldTheory/Galois/Basic.lean

```diff
@@ -278,6 +278,73 @@ def galoisCoinsertionIntermediateFieldSubgroup [FiniteDimensional F E] [IsGalois
 
 end IsGalois
 
+section
+
+/-In this section we prove that the normal subgroups correspond to the Galois subextensions
+in the Galois correspondence and its related results.-/
+
+variable {K L : Type*} [Field K] [Field L] [Algebra K L]
+
+open IntermediateField
+
+open scoped Pointwise
+
+lemma AlgEquiv.restrictNormalHom_ker (E : IntermediateField K L) [Normal K E]:
+    (restrictNormalHom E).ker = E.fixingSubgroup := by
```

Comment:
Would `IntermediateField.restrictNormalHom_ker` be a better name, to enable dot-notation?